Add section "Image scales from brain" to Classic UI > Image handling (#1536)
plone/documentation: docs/classic-ui/images.md

Please note that this example has the `resolve_uid_and_caption` filter disabled to see the scale names better.
The real `src` URLs look more like `http://localhost:8080/Plone50/dsc04791.jpg/@@images/778f9c06-36b0-485d-ab80-12c623dc4bc3.jpeg`.
```

## Image scales from catalog brain

> For all `NamedBlobImage` fields, we can get existing scale info's directly from the catalog brain.

Reviewer: ```suggestion
For all `NamedBlobImage` fields, we can get existing scale information directly from the catalog brain.
```

> Given a content type with a `NamedBlobField` named `picture`, we can get the following info's, by calling the `image_scales` attribute on the catalog brain.

Reviewer: ```suggestion
Given a content type with a `NamedBlobField` named `picture`, we can get the following information by calling the `image_scales` attribute on the catalog brain.
```

                                     'width': 16},
                         'mini': {'download': '@@images/picture-200-3de96ae4288dfb18f5589c89b861ecc1.jpeg',
                                  'height': 200,
                                  'width': 200},
                         'preview': {'download': '@@images/picture-400-60f60942c8e4ddd7dcdfa90527a8bae0.jpeg',
                                     'height': 400,
                                     'width': 400},
                         'teaser': {'download': '@@images/picture-600-1ada88b8af6748e9cbe18a34c3127443.jpeg',
                                    'height': 600,
                                    'width': 600},
                         'thumb': {'download': '@@images/picture-128-80fce253497f7a745315f58f3e8f3a0c.jpeg',
                                   'height': 128,
                                   'width': 128},
                         'tile': {'download': '@@images/picture-64-220d6703eac104c59774a379a8276e76.jpeg',
                                  'height': 64,
                                  'width': 64}},
              'size': 238977,
              'width': 800}]}
```

> In these info's we have everything we need to generate our image URL's, without waking up any object's.

Reviewer: ```suggestion
This information shows we have everything we need to generate our image URLs, without waking up any objects.
```
